### ISSUE TITLE
Possible bug with wrapperSetTitle

### ISSUE THREAD

**Issue by anoadragon453** [HelloZeroNet/ZeroNet#1792]
### Step 1: Please describe your environment

  * ZeroNet version: d3bf001752b5df11ebca5ef60f3974599d5c38bf

### Step 2: Describe the problem:

Setting the title via:

```javascript
ZeroFrame.cmd('wrapperSetTitle', 'My Great Title');
```

works fine, but takes in a string as input. Whereas you would also expect the following to work:

```javascript
ZeroFrame.cmd('wrapperSetTitle', {title: 'My Great Title'});
```

But not so, as the title is set to `[object Object]`.

Presumably every ZeroFrame command should handle the object syntax, including `wrapperSetTitle`?

**Comment by HelloZeroNet** [comment]
Yes, this is a known error. It happens because all command starts with "wrapper" is handled by javascript that does not support positional arguments like python does. Right now I have no easy solution for this.
